dart/math/Geometry.h

```diff
@@ -533,6 +533,35 @@ Eigen::Vector2d computeClosestPointOnSupportPolygon(
     const Eigen::Vector2d& _p,
     const SupportPolygon& _support);
 
+
+// Represents a bounding box with minimum and maximum coordinates.
+class BoundingBox {
```

Comment:
I'd prefer if this class had AABB in the name, since some algorithms (e.g. inside FCL) also use OBBs.